cloud-functions/functions/scale_up/scale_up.go

```diff
@@ -62,9 +62,14 @@ func CreateInstance(ctx context.Context, project, zone, template, instanceName,
 	gpgcheck=0
 	EOL
 	fi
-	sudo yum -y update
 
-	sudo yum install -y jq
+	os=$(awk -F= '/^NAME/{print $2}' /etc/os-release)
+    if [[ "$os" = *"Rocky"* ]]; then
+		sudo yum install -y jq bc
```

Comment:
we always want to install jq
what are bc and perl-interpreter for?
Do we have to include kernel-devel?

Reply:
I need perl-interpreter is one of the kernel-devel dependency
Also, I need to bc we use it to calculate OBS `tiering_percent=$(echo "$full_capacity * 100 / $OBS_TIERING_SSD_PERCENT" | bc)` 

And I used rpm URL to make sure to install the right version